<!-- 

-->
<launch>
    <env name="ROSCONSOLE_CONFIG_FILE"
       value="$(find frontier_detector)/config/ffd_rosconsole.conf"/>
    <param name="use_sim_time" value="false"/> <!--by kmHan-->  
    
    <arg name="rviz_config" value ="$(find frontier_detector)/launch/includes/rviz_config/frontier_detector.rviz"/>
    
    <include file="$(find frontier_detector)/launch/includes/move_fetch.launch">
    </include> 
    <param name="frame_id" type="string" value="map" />
    <param name="num_downsamples" value="1" />
<!--    <env name="ROSCONSOLE_CONFIG_FILE"
    value="$(find frontier_detector)/config/ffd_rosconsole.config"/>-->

	<node pkg="frontier_detector" type="frontier_detector_node" name="frontier_detector" output="screen">

    </node>
       
    <!--node name="rviz" pkg="rviz" type="rviz" required="true" args="-d $(arg rviz_config)"/-->

        
    <!--/group-->
</launch>
